src/ui/views/meter.rs

```diff
@@ -0,0 +1,470 @@
+use vizia::vg::{Color, Paint, Path};
+use vizia::*;
+
+/// The direction the meter bar shows the peak in.
+/// The semantic is LowToHigh, so DownToUp is the standard vertical meter design
+///
+/// This is also used to decide the orientation of the meter
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Data)]
+pub enum Direction {
+    /// The standard vertical meter direction
+    DownToUp,
```

Comment:
Is there a reason why these have to be "x to y" instead of just "y"? Since the enum is called `Direction`, I feel like it's clear that `Up` means "bottom upwards" and `Right` means "from left to right"

Reply:
I don't think there was a proper reason aside from me using it as the first intuition. Just the directions alone should be clear enough. I'll put it in a commit at the end